SConstruct

```diff
@@ -190,16 +200,50 @@ if unknown:
     for item in unknown.items():
         print("    " + item[0] + "=" + item[1])
 
+# Compatibility with old bits argument on Windows/Linux.
+# Not needed on macOS or web. Android and iOS don't use env["arch"].
+if env["platform"] in ["windows", "linux"]:
+    if env["bits"] == "64":
+        env["arch"] = "x86_64"
+    elif env["bits"] == "32":
+        env["arch"] = "x86_32"
+
+# Process CPU architecture argument.
+if env["arch"] == "auto":
```

Comment:
Since there's no platform-specific `get_opts()` here you should likely also set the expected defaults for Android and iOS (arm64).

Reply:
Removed this bit of code. I made it so that Android and iOS are not affected by this argument to avoid breaking too many things. I guess it could still be changed in a follow-up, but this PR I'm focusing on desktop and web.